dist/ads.js

```diff
@@ -84,9 +90,9 @@ const parseAdsTxt = async (response) => {
     // Count unique and remove domain Sets for now
     for (let accountType of Object.values(result.account_types)) {
       accountType.domain_count = accountType.domains.size;
-      delete accountType.domains // Keeping a list of domains may be valuable for further research, e.g. accountType.domains = [...accountType.domains];
+      accountType.domains = Array.from(accountType.domains); // delete accountType.domains
```

Comment:
@max-ostapenko why did you delete this before despite saying it might be valuable? Is it OK to add them back in?

Reply:
@tunetheweb It adds quite some data, so I expected to bring it as a second step.
And now we have a clear use case for this data in WebAlmanac - exploring ads supply  transparency and transparency.

Reply:
Hmmm I see what you mean. That is quite a bit of data. WDYT @rviscomi ?

Reply:
If the data is useful, LGTM. I'd imagine this only affects a fraction of sites in the dataset, but let's run it for one month and roll it back if we see it significantly affects the size of the crawl.